### DISCUSSION TITLE
Retrieving Dynamic fields

### DISCUSSION THREAD

**Discussion by genzerstech** [milvus-io/milvus#35561] [Q&A and General discussion]
I need to move one collection from source milvus database to destination milvus database. while we achieved everything we wanted, the dynamic fields are not being able to be retrieved from the source collection. How to retrieve the dynamic field json or keys and values for a given document in the collection?

Or is there a better way to move certain set of collections from source milvus database to destination milvus database.

**Comment by xiaofan-luan** [comment]
what is the way yo are using to pull data out of your source milvus?

@Thomas-HuWei @yelusion2 might be able to help
